src/smc_sagews/smc_sagews/sage_salvus.py

```diff
@@ -3571,3 +3571,302 @@ def help(*args, **kwds):
 
 - **License information:** For license information about Sage and its components, enter `license()`."""%sage.version.version
         salvus.md(s)
+
+
+"""
+sage_jupyter.py
+
+Spawn and send commands to jupyter kernels.
+"""
+
+#########################################################################################
+#       Copyright (C) 2016, SageMath, Inc.                                              #
+#                                                                                       #
+#  Distributed under the terms of the GNU General Public License (GPL), version 2+      #
+#                                                                                       #
```

Comment:
Can you add an `AUTHOR: Hal Snyder` thing as well, like we do with Sage.  It's useful to easily know who wrote the first version of something...